.github/release-please/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "separate-pull-requests": true,
+  "group-pull-request-title-pattern": "chore: release ${component} ${branch}",
+  "bootstrap-sha": "a980ac61f484a3b6bdbe11e606977d0d92d4e9ff",
```

Comment:
Don't forget to point it to the latest commit before merging

Reply:
Will do

Reply:
I've already did.